src/lexedata/edit/add_concepticon.py

```diff
@@ -126,10 +126,37 @@ def add_concepticon_references(
     dataset.write(ParameterTable=write_back)
 
 
+def add_concepticon_definitions(
+    dataset: pycldf.Dataset,
+    column_name: str = "Concepticon_Definition",
+):
+    # Create a concepticon_definition column
+    try:
+        dataset.add_columns("ParameterTable", column_name)
+        dataset.write_metadata()
+    except ValueError:
```

Comment:
As it is now, there's no way to add/update Concepticon definitions in a dataset that already has them (partially or outdated).
I think it would be acceptable to overwrite existing definitions (they should not be edited in the dataset, but are just cached according to the concepticonReference, after all), so if this ValueError is due to the column already existing, INFO that existing concepticon definitions are overwritten and proceed, instead of dying.